src/components/common/CourtOrderPoa.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <section id="court-order-poa">
+    <header>
+      <h2>{{sectionNumber}} Court Order and Plan of Arrangement</h2>
+      <div class="py-4">
+        If this filing is pursuant to a court order, enter the court order number. If this
+        filing is pursuant to a plan of arrangement, enter the court order number and select
+        Plan of Arrangement.
+      </div>
+    </header>
+    <div :class="{ 'invalid-section': invalidCourtOrder }">
+      <CourtOrderPoaShared
+        id="court-order"
+        :autoValidation="getAppValidate"
+        :hasDraftPlanOfArrangement="hasDraftPlanOfArrangement"
+        :invalidSection="invalidCourtOrder"
```

Comment:
This prop value is not declared anywhere.

In [Alteration.vue](https://github.com/bcgov/business-edit-ui/blob/0dd1590dc82f7f29c83dbcf41c80b807d7144e6b/src/views/Alteration.vue#L205), this value is a local getter that considers a couple of flags in the store. I believe the same should exist here.

Reply:
To be clear, I think this component needs both the Store getter and the local getter,

Reply:
obsolete comment